task_templates/2_estimators/11_python_huggingface_vit/custom.py

```diff
@@ -30,7 +30,13 @@ def __len__(self):
 
 
 class CustomTask(BinaryEstimatorInterface):
-    def fit(self, X, y, row_weights=None, **kwargs):
+    def fit(self, X, y, row_weights=None, parameters=None, **kwargs):
```

Comment:
If you want `parameters` to be required param, maybe make it positional t.e required?!

Reply:
Actually,
maybe just pass `epochs` directly as `kwargs` and check it is not None. i.e.
```
epochs = kwargs.get("epochs")
if epochs is None:
   raise ValueError("epoch has to be provided in kwargs and can not be None")
```

Reply:
I believe all hyperparameters are added to parameters, they are not accessible individually.  This is passed by drum from reading the model-metadata

Reply:
oh, I see, 
ok then